examples/operator/operator.yaml

```diff
@@ -5432,7 +5432,7 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: vitess-operator
-          image: planetscale/vitess-operator:latest
+          image: planetscale/vitess-operator:v2.6.0
```

Comment:
Same question here.

Reply:
No, not yet: https://hub.docker.com/r/planetscale/vitess-operator/tags

We're working on that here: https://github.com/planetscale/vitess-operator/pull/253

We have to make a number of API related changes in the operator for Vitess 13.0 (calls moved form wrangler to vtctldgrpcserver etc).